src/containers/shared/components/Transaction/SignerListSet/Description.tsx

```diff
@@ -17,7 +17,7 @@ export const Description = ({
         {t('set_signer_list_description', { quorum: tx.SignerQuorum })}:
       </div>
       <ul className="signers">
-        {tx.SignerEntries?.map((d) => (
+        {tx.SignerEntries?.map((d: any) => (
```

Comment:
Can you give `d` a more descriptive name?

Reply:
On my local this is derived fine based in SignerListSet passed as the type above.  We can manually specify the type if SignerEntry from types.ts